docs/source/about_mapstyle_vs_iterable.mdx

```diff
@@ -205,6 +205,43 @@ for epoch in range(n_epochs):
         pass
 ```
 
+## Batching data in IterableDataset
```

Comment:
this can go in stream.mdx instead ?

Btw in the future it would be cool to have `Dataset.batch()` as well :)